joint_names:
  - joint1
  - joint2
  - joint3
  - joint4
  - joint5
  - joint6
  - joint7

robot_ip: 127.0.0.1
franka_states_publish_rate: 30.0  # [Hz]

